### PR TITLE
fix safari scroll

### PR DESCRIPTION
**Pull request by maskin25**
# What this PR does

Fix  scroll in Safari

## Which issue(s) this PR fixes

https://github.com/grafana/oncall/issues/415

## Checklist

- [ ] Unit, integration, and e2e (if applicable) tests updated
- [ ] Documentation added (or `pr:no public docs` PR label added if not required)
- [x] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

**Review by iskhakov — APPROVED** [review]

**Review by joeyorlando — CHANGES_REQUESTED** [review]
can you uncomment out the Safari browser in the e2e tests [here](https://github.com/grafana/oncall/blob/dev/grafana-plugin/playwright.config.ts#L68-L74)?

**Comment by maskin25** [comment]
@joeyorlando I enabled tests on Safari

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Addressed bug with iOS mobile push notifications always being set to critical by @imtoori and @joeyorlando ([#1646](https://github.com/grafana/oncall/pull/1646))
 - Fixed issue where Viewer was not able to view which people were oncall in a schedule ([#999](https://github.com/grafana/oncall/issues/999))
 - Fixed a bug with syncing teams from Grafana API by @vadimkerr ([#1652](https://github.com/grafana/oncall/pull/1652))
+- Fixed bug when web schedules/shifts use non-UTC timezone and shift is deleted by @matiasb ([#1661](https://github.com/grafana/oncall/pull/1661))
```
Comment (joeyorlando) [inline CHANGELOG.md:31]: can you remove this line?

File: CHANGELOG.md [hunk 2]
```diff
@@ -28,6 +28,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Enable tests on Safari
+- Fix Safari scroll ([#415](https://github.com/grafana/oncall/issues/415))
```
Comment (joeyorlando) [inline CHANGELOG.md:33]: these two lines should be moved to an "Unreleased" section

File: .github/workflows/linting-and-tests.yml [hunk 3]
```diff
@@ -377,11 +377,10 @@ jobs:
 
       - name: Install Playwright binaries/dependencies
         if: steps.playwright-cache.outputs.cache-hit != 'true'
-        # if more browsers are added, will need to modify the "npx playwright install" command
         # https://stackoverflow.com/questions/65900299/install-single-dependency-from-package-json-with-yarn
         run: |
           yarn add "@playwright/test@${{ env.PLAYWRIGHT_VERSION }}"
-          npx playwright install --with-deps chromium firefox
+          npx playwright install --with-deps chromium firefox webkit
```
Comment (joeyorlando) [inline .github/workflows/linting-and-tests.yml:383]: needed to install webkit browser on ci

File: grafana-plugin/integration-tests/utils/forms.ts [hunk 4]
```diff
@@ -39,7 +39,7 @@ export const clickButton = async ({
   dataTestId,
 }: ClickButtonArgs): Promise<void> => {
   const baseLocator = dataTestId ? `button[data-testid="${dataTestId}"]` : 'button';
-  const button = (startingLocator || page).locator(`${baseLocator} >> text=${buttonText}`);
+  const button = (startingLocator || page).locator(`${baseLocator}:not([disabled]) >> text=${buttonText}`);
```
Comment (joeyorlando) [inline grafana-plugin/integration-tests/utils/forms.ts:42]: this _should_ make some of the flaky schedules tests more reliable.. I think the issue was that the tests were trying to click on the "Add rotation" button, which according to this screenshot, starts off `disabled`. By adding this, Playwright _should_ wait until the desired button is enabled before trying to click it:
Screenshot 2023-03-30 at 16 12 25

**Review by joeyorlando — APPROVED** [review]

**Pull request merged**